test/features/create_organization_test.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
+
+class CreateOrganizationTest < Capybara::Rails::TestCase
+
+  def organization
+    @organization ||= organizations :mapc
+  end
+  alias_method :org, :organization
+
+  def setup
+    @user = users :normal
+    @user.password = 'password'
+
+    @unauthorized_user = users :lower_case
+    @unauthorized_user.password = 'drowssap'
+  end
+
+  test "signed out guest, visit organization creation, and be redirected" do
+    visit new_organization_path
+    assert_content page, 'Log in'
+  end
+
+  test "signed in user, visit organization creation, and not be redirected" do
+    sign_in @user, visit: true, submit: true
+    visit new_organization_path
+    assert_content page, 'Create Organization'
+  end
+
+  test "signed in user visits new organization path, and successfully creates organization" do
+    sign_in @user, visit: true, submit: true
+    visit new_organization_path
+    fill_in 'organization_name', :with => 'Boston Properties'
```

Comment:
The block of `fill_in` methods for each example seem to be the same between all organizations, which signals that this is ready to be refactored. Write a helper similar to [the sign_in helper](https://github.com/MAPC/developmentdatabase/blob/develop/test/support/session_helpers.rb#L15), but don't include the `visit` method in the helper so it's clear in the test itself which path you're visiting.